go/vt/vtgate/planbuilder/symboltable.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2016, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package planbuilder
+
+import (
+	"errors"
+
+	"github.com/youtube/vitess/go/vt/sqlparser"
+)
+
+// SymbolTable contains the symbols for a SELECT
```

Comment:
I didn't see your answer to my previous comment. So I'm repeating it here:
"Define what you mean by a symbol. It looks like you’re only storing table names."